cmd/ocp-certificate-api/main.go

```diff
@@ -23,14 +27,14 @@ import (
 // buildDB - init db (postgres)
 func buildDB() *sqlx.DB {
```

Comment:
А, кстати, в прошлый раз-то не заметил
```suggestion
func database() *sqlx.DB {
```
или
```suggestion
func initDB() *sqlx.DB {
```